src/handler.rs

```diff
@@ -17,7 +16,7 @@ use std::{future::Future, net::IpAddr, net::SocketAddr, path::PathBuf, sync::Arc
     feature = "compression-zstd",
     feature = "compression-deflate"
 ))]
-use crate::compression;
+use {crate::compression, headers::HeaderValue};
```

Comment:
Please restore this change and line 9 above too.  It is because I prefer to distinguish between local and external crates.

Btw I will see if I can enforce this via some lint rule or so.

Reply:
Done.